spec/javascripts/components/notes/note_spec.js

```diff
@@ -0,0 +1,64 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NoteComponent from 'components/notes/Note';
+import NoteForm from 'components/notes/NoteForm';
+import Note from 'models/note.js';
+
+const note = new Note({note: 'Test Note #1'});
+
+describe('<Note />', function() {
+
+  beforeEach(function() {
+    jasmineEnzyme();
+    sinon.stub(I18n, 't');
+    sinon.stub(window.md, 'makeHtml');
+  });
+
+  afterEach(function() {
+    I18n.t.restore();
+    window.md.makeHtml.restore();
+  });
+
+  it("should have its content parsed", function() {
+    const wrapper = shallow(<NoteComponent note={note} />);
+    expect(window.md.makeHtml).toHaveBeenCalled();
```

Comment:
I think it would be good to assert the value that was passed to `makeHtml` as well. You can do that using `toHaveBeenCalledWith(expectedParameter)` instead of simply `toHaveBeenCalled()`.